File: O curso definitivo de Lógica de Programação. Do VisualG às principais linguagens do mercado C, C++, Python, C# e Java/C/ATIVIDADES/retangulo.c
#include <stdio.h>
#include <math.h>


int main(){
    float altura,base,area,perimetro,diagonal;

    printf("Base do retangulo: ");
    scanf("%f", &base);

    printf("Altura do retabgulo: ");
    scanf("%f",&altura);

    area = base * altura;

    printf("AREA = %4.f/n",area);

    perimetro = 2 * (base + altura);

    printf("perimetro = %.4f\n", perimetro);

    diagonal = sqrt(base * base + altura * altura);

    printf("diagonal = %.4f\n", diagonal);

    return 0;
    

}

File: O curso definitivo de Lógica de Programação. Do VisualG às principais linguagens do mercado C, C++, Python, C# e Java/JAVA/Atividaes/retangulo.java

import java.util.Scanner;
import java.util.Locale;

public class retangulo {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		
		double area,base,altura,perimetro,diagonal;
		
		System.out.print("Digite a base do retangulo: ");
		base = sc.nextDouble();
		System.out.print("Digite a altura do retangulo: ");
		altura = sc.nextDouble();
		
		area = base * altura;
		perimetro = 2 * (base + altura);
		
		diagonal = Math.sqrt(base*base + altura * altura);
		
		System.out.println("AREA = " + String.format("%.4f",area) );
		System.out.println("PERIMETRO = " + String.format("%.4f",perimetro));
		System.out.println("DIAGONAL = " +  String.format("%.4f",diagonal));
		
		sc.close();
		
		

	}

}